packages/text-bitmap/src/utils/extractCharCode.ts

```diff
@@ -0,0 +1,4 @@
+export function extractCharCode(str: string): number
```

Comment:
```suggestion
/**
 * Ponyfill for IE because it doesn't support `codePointAt`
 * @private
 */
export function extractCharCode(str: string): number
```